packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -167,11 +167,13 @@ export default async function initSanity(
       print(`Coupon "${intendedCoupon}" validated!\n`)
     } catch (err) {
       if (err.statusCode == '404') {
-        const useDefaultPlan = await prompt.single({
-          type: 'confirm',
-          message: `Coupon "${intendedCoupon}" is not available, use default plan instead?`,
-          default: true,
-        })
+        const useDefaultPlan =
+          unattended ??
+          (await prompt.single({
+            type: 'confirm',
+            message: `Coupon "${intendedCoupon}" is not available, use default plan instead?`,
+            default: true,
+          }))
```

Comment:
Can we print a warning if unattended?
```suggestion
          }))
          
          if (unattended) {
            output.warn(`Coupon "${intendedCoupon}" is not available - using default plan`)
          }
```

Reply:
Yep! Thank you. I've added and tested the suggested warnings.